java/code/src/com/redhat/rhn/domain/channel/Channel.java

```diff
@@ -70,6 +75,113 @@
 @Entity
 @Table(name = "rhnChannel")
 @Inheritance(strategy = InheritanceType.JOINED)
+@NamedQueries({
```

Comment:
Not sure if this annotation is needed with 6.5.
It could be possible to have just multiple NamedQuery annotation